payments-core/src/main/java/com/stripe/android/payments/PaymentFlowResult.kt

```diff
@@ -102,6 +102,6 @@ sealed class PaymentFlowResult {
 
         internal val sourceId: String? = null,
         internal val source: Source? = null,
-        internal val stripeAccountId: String? = null
+        val stripeAccountId: String? = null
```

Comment:
Not familiar with our Android code, does this make sense if the class is `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)`? I wanted to check in the test that the result had the correct stripeAccountId, but I don't want to expose it publicly.

Reply:
In practice merchants can't access this because the parent class is annotated with restrict to. But the nested class `Unvalidated` isn't -- which is what's causing the tool to add it to the api file. You can safely add the restrict to annotation to the nested `Unvalidated` class, and avoid it being added to the api file.